modules/nf-core/deepvariant/postprocessvariants/main.nf

```diff
@@ -0,0 +1,77 @@
+process DEEPVARIANT_POSTPROCESSVARIANTS {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
The suggestion was to change from `process_single` to  `process_high` because the new Deepvariant postprocessvariants is multithreaded. I've changed to `process_medium` because it seems only part of it is multithreaded

Reply:
Sounds reasonable!